tasks/deploy.yaml

```diff
@@ -25,12 +18,11 @@ tasks:
       - description: "Deploy the TLS certs for Istio"
         cmd: |
           npm ci
-          npx --yes ts-node examples/tls-certs.ts
+          npx --yes ts-node bundles/tls-certs.ts
 
   - name: single-capability
     actions:
       - description: "Deploy the requested capability Zarf Package (must set CAPABILITY environment variable)"
         cmd: |
-          export ARCH=$(uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/')
-          zarf package deploy build/zarf-package-uds-capability-${CAPABILITY}-${ARCH}.tar.zst --confirm
-          zarf package deploy build/zarf-package-pepr-uds-core-${ARCH}-*.tar.zst --confirm || true
+          zarf package deploy build/zarf-package-uds-capability-${CAPABILITY}-${UDS_ARCH}.tar.zst --confirm
+          zarf package deploy build/zarf-package-pepr-uds-core-${UDS_ARCH}-*.tar.zst --confirm || true
```

Comment:
Is UDS_ARCH coming as built in from UDS as the current system ARCH or am i missing something about where its defined?

Reply:
if the former is the case i have no issues with this PR, if the latter is the case i'd like to understand, but otherwise i think this all looks good. 😄

Reply:
UDS_ARCH was added a couple versions ago to provide the zarf-compatible binary arch (amd64 or arm64) for this exact usecase.